harness/main.go

```diff
@@ -15,17 +15,30 @@ func MakeApp(opts *ExporterOpts) *cli.App {
 	app.UsageText = opts.Usage
 	app.Action = exp.main
 	app.Flags = append(opts.Flags,
-		cli.IntFlag{
-			Name:  "port",
-			Usage: "The port number used to expose metrics via http",
-			Value: 7979,
-		},
 		cli.StringFlag{
 			Name:  "log-level",
 			Usage: "Set Logging level",
 			Value: "info",
 		},
 	)
+
+	var portFlagExists bool
```

Comment:
I think this could be implemented much generally.
Something like by merging two options set `opts.Flags` and options given here.

1. `defaultOpts := make(map[string]*cli.Flag)`
2. `app.Flags = buildOptsWithDefault(opts.Flags, defaultOpts)`